.pre-commit-config.yaml

```diff
@@ -112,18 +112,20 @@ repos:
       - id: debug-statements
       - id: name-tests-test
         args: [--pytest-test-first]
-  - repo: https://github.com/biomejs/pre-commit
-    rev: v0.1.0
+  - repo: local
     hooks:
       - id: biome-check
         name: biome (javascript, typescript, json)
-        additional_dependencies: ['@biomejs/biome@1.5.3']
+        entry: biome check .
+        language: system
+        types: [text]
+        files: "\\.(jsx?|tsx?|c(js|ts)|m(js|ts)|d\\.(ts|cts|mts)|jsonc?)$"
```

Comment:
this provides a worse user experience because it requires the developer to have everything else set up.

if pre-commit manages the tools it will always be in the right state and not whatever the developer has (messed up).  `language: system` and `repo: local` are unsupported escape hatches and miss the point of `pre-commit`

the rest of the file (incorrectly) uses `repo: local` and I'd like to fix that as well eventually

Reply:
I disagree but I removed the proposed changes. I don't think I should install Biome twice (100mb x 2).

Can you re-review it again?